### PR TITLE
BUGFIX: RAIL-1044 Fix computing paging count and 1 dimension merging in executeAfm

### PR DESCRIPTION
**Pull request by ocetnik**
`6.1.1-ocetnik-doc-paging-count-2018-07-12T10-46-28-143Z`

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/33/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/27/ : SUCCESS in 2m 40s

File: src/execution/execute-afm.ts [hunk 1]
```diff
@@ -202,14 +202,20 @@ export function mergePage(
     const result = prevExecutionResult;
     const { headerItems, data, paging } = executionResult;
 
+    function mergeHeaderItemsForEachAttribute(dimension: number) {
```
Comment (martinmosko) [inline src/execution/execute-afm.ts]: please move this outside mergePage function and avoid of using function side-effects (modifying result object)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/34/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/28/ : SUCCESS in 1m 37s

**Comment by martinmosko** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/10/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/29/ : SUCCESS in 1m 42s

**Pull request merged**